vassal-app/src/main/java/VASSAL/build/module/GlobalOptions.java

```diff
@@ -217,6 +233,21 @@ public boolean isUseClassicMoveFixedDistance() {
   public void setUseClassicMoveFixedDistance(boolean b) {
     useClassicMoveFixedDistance = b;
   }
+  
+  
+  public boolean isPrefMacLegacy() {
```

Comment:
It's funny I had them precisely so, and then changed them so that it was clear that this is a different stage from the internal SwingUtils being set (and I didn't want SwingUtils to have to be calling back up into this). At the time I changed it, there was the possibility I was going to need this preference to be a "Requires Restart" preference ... so maybe this made more sense in that case. I personally think it's got maximum clarity in its current form, but if you still prefer I change it let me know.